### PR TITLE
minor: Bump react-components version to 9.30.0

### PR DESCRIPTION
**Pull request by FengQiMS**
Bump react-components version to 9.30.0

### CODE REVIEW

File: packages/nx-plugin/src/generators/library/generator.spec.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('create-package generator', () => {
     const pkgJson = readJson(tree, paths.packageJson);
     expect(pkgJson.peerDependencies).toMatchInlineSnapshot(`
       {
-        "@fluentui/react-components": ">=9.25.1 <10.0.0",
+        "@fluentui/react-components": ">=9.35.1 <10.0.0",
```
Comment (FengQiMS) [inline packages/nx-plugin/src/generators/library/generator.spec.ts:34]: Does this matter, it was automatically updated by running test

File: yarn.lock [hunk 2]
```diff
@@ -1296,752 +1296,804 @@
...
-    "@fluentui/react-theme" "^9.1.9"
-    "@fluentui/react-utilities" "^9.10.1"
-    "@griffel/react" "^1.5.7"
-    "@swc/helpers" "^0.4.14"
-
-"@fluentui/react-icons@^2.0.203":
-  version "2.0.204"
-  resolved "https://registry.yarnpkg.com/@fluentui/react-icons/-/react-icons-2.0.204.tgz#cc93486e1cbea5f0df2cd6b1648d47fe6eb06557"
-  integrity sha512-zl1T5CorbJ3cLeM3BfXxT1Bdm1f56gTXagrJG+f+b3gs+bI5qeNmCkZv5yq0DVdWx5KAqYoeX57wkn7GM5GyTA==
+"@fluentui/keyboard-keys@^9.0.6":
```
Comment (ling1726) [inline yarn.lock:1299]: can you please run

`npx yarn-deduplicate`

in your repo root and check in the changes to yarn.lock? I haven't had the time to do some automation checks on that yet unfortunately

Comment (FengQiMS) [reply yarn.lock:1299]: resolved, thanks

File: packages/nx-plugin/src/generators/library/generator.spec.ts (hunk 1 above)
Comment (ling1726) [reply packages/nx-plugin/src/generators/library/generator.spec.ts:34]: this is fine, all new packages creatd after this PR will have a higher lower bound for compatibility. It's overridable by each package after autogeneration in any case

**Pull request merged**